src/NHapi.SourceGeneration/TableRepository.cs

```diff
@@ -1,94 +1,63 @@
+  of this file under the MPL, indicate your decision by deleting  the provisions above
+  and replace  them with the notice and other provisions required by the GPL License.
+  If you do not delete the provisions above, a recipient may use your version of
+  this file under either the MPL or the GPL.
 */
 
-using System;
-
 namespace NHapi.SourceGeneration
 {
-   /// <summary> A place where table keys and values are stored.  This may be implemented
-   /// with a database, an LDAP directory, local RAM, etc.  At a minimum, any
-   /// underlying repository must supply the values for standard HL7 tables.
-   /// Site-defined tables may also be supported.
-   /// </summary>
-   /// <author>  Bryan Tripp (bryan_tripp@sourceforge.net)
-   /// </author>
-   public abstract class TableRepository
-	{
-		/// <summary> Returns a TableRepository object.</summary>
-		public static TableRepository Instance
-		{
-			get
-			{
-				if (rep == null)
-				{
-					//currently using DBTableRepository ... 
-					rep = new DBTableRepository();
-				}
-
-				return rep;
-			}
-		}
-
-		/// <summary> Returns a list of HL7 tables.  </summary>
-		public abstract int[] Tables { get; }
-
-		private static TableRepository rep = null;
+    /// <summary> A place where table keys and values are stored.  This may be implemented
+    /// with a database, an LDAP directory, local RAM, etc.  At a minimum, any
+    /// underlying repository must supply the values for standard HL7 tables.
+    /// Site-defined tables may also be supported.
+    /// </summary>
+    /// <author>  Bryan Tripp (bryan_tripp@sourceforge.net).
+    /// </author>
+    public abstract class TableRepository
+    {
+        private static TableRepository rep = null;
+
+        /// <summary> Returns a TableRepository object.</summary>
+        public static TableRepository Instance
+        {
+            get
+            {
+                rep = rep ?? new DBTableRepository();
+                return rep;
+            }
+        }
 
-		/// <summary> Returns true if the given value exists in the given table.</summary>
-		public abstract bool checkValue(int table, String value_Renamed);
+        /// <summary> Returns a list of HL7 tables.  </summary>
+        public abstract int[] Tables { get; }
 
-		/// <summary> Returns a list of the values in the given table. </summary>
-		public abstract String[] getValues(int table);
+        /// <summary> Returns true if the given value exists in the given table.</summary>
+        public abstract bool CheckValue(int table, string value_Renamed);
 
-		/// <summary> Returns the value corresponding to the given table and key.</summary>
-		/// <throws>  UnknownValueException if the value can not be found.  This may be an UnknownTableException.   </throws>
-		public abstract String getDescription(int table, String value_Renamed);
+        /// <summary> Returns a list of the values in the given table. </summary>
+        public abstract string[] GetValues(int table);
```

Comment:
it looks like you didnt leave the `getValues(int table)`  or `checkValue` or `tables` in place with an obsolete attribute?
is that an issue?

Reply:
ah never mind its because its saurce generation 😁 there is no need.

Reply:
I think I asked you about this. As this is not part of the nuget package, even thoght they are public members, they shouldn't break any changes outside of the repo - I've fixed all references (if there were any) in both solutions. We should probably do a review of what can be made internal within each project. Happy to resolve this?